src/test/java/uk/gov/hmcts/reform/sscs/personalisation/WithRepresentativePersonalisationTest.java

```diff
@@ -15,20 +15,20 @@
 import uk.gov.hmcts.reform.sscs.config.AppConstants;
 
 @RunWith(JUnitParamsRunner.class)
-public class AppealLapsedPersonalisationTest {
+public class WithRepresentativePersonalisationTest {
 
     private static final String CASE_ID = "54321";
 
     private SscsCaseData response;
 
-    private AppealLapsedPersonalisation appealLapsedPersonalisation =
-        new AppealLapsedPersonalisation();
+    private WithRepresentativePersonalisation withRepresentativePersonalisation =
+        new WithRepresentativePersonalisation();
 
     @Test
     @Parameters(method = "generateSscsCaseDataForTest")
     public void givenSyaAppealCreated_shouldSetRepresentativeNameIfPresent(
```

Comment:
please, rename this method since the given condition makes reference to the SyaAppealCreated event. Something more generic such as givenSscsCaseDate_should... makes more sense, right?

Reply:
done